src/gui/addnewtorrentdialog.ui

```diff
@@ -127,6 +127,13 @@
         </property>
        </widget>
       </item>
+      <item row="1" column="1">
+       <widget class="QCheckBox" name="default_label">
```

Comment:
Although you may found the names of the different species in the existing code, in new code you should use only camelCased names.